samples/zigbee/light_bulb/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+endif()
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}.overlay")
+  set(DTC_OVERLAY_FILE "${DTC_OVERLAY_FILE} ${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}.overlay")
+endif()
```

Comment:
This will not work with when using deprecated board names nor aliased board names.
Please remove.

Board overlays are handled in Zephyr build system, when named correctly.
See https://github.com/NordicPlayground/fw-nrfconnect-zephyr/blob/master/cmake/app/boilerplate.cmake#L463-L471

Reply:
The overlay files have been combined into one and renamed to app.overlay